btyaccpa.ske

```diff
@@ -194,8 +194,8 @@ static Yshort *yylexemes=0;
 int yyparse(void);
 int yylex(void);
 
-static void YYSCopy(YYSTYPE *to, YYSTYPE *from, int size) {
-  int i;                             
+static void YYSCopy(YYSTYPE *to, YYSTYPE *from, size_t size) {
+  size_t i;
```

Comment:
`size_t` is an unsigned type, so this makes the following loop loop forever...

Reply:
Ah yup silly mistake, interesting that I didn't hit this running a parser built with it. Fixed by using ptrdiff_t here.